highcharts/highcharts.d.ts

```diff
@@ -3249,11 +3254,19 @@ interface HighchartsLineStates {
 
 interface HighchartsBarStates {
     /**
+     * A specific border color for the hovered point. Defaults to inherit the normal state border color.
+     */
+    borderColor?: string | HighchartsGradient;
```

Comment:
I have introduced the `Color` type in a later commit, as you could see in #11506